src/main/java/core/mate/academy/model/TruckProducer.java

```diff
@@ -0,0 +1,11 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        return List.of(new Truck());
```

Comment:
Consider creating a local variable for the list of machines before returning it. This can make debugging easier if you need to inspect the list before it is returned or if additional processing is required in the future.